.npmrc

```diff
@@ -1 +1,2 @@
 @yalesites-org:registry=https://npm.pkg.github.com
+//npm.pkg.github.com/:_authToken=${YALESITES_BUILD_TOKEN}
```

Comment:
Leaving a reminder to add instructions around this in the README. Does not need to be a part of this PR.